docs/user/exercises/programming.rst

```diff
@@ -57,6 +57,13 @@ The following sections list the rules that are active for the default static cod
 
 .. include:: programming-sca-default-configuration.inc
 
+Submission Policy Configuration
+-------------------------------
+
+The following sections explains the configuration options for submission policies
+
+.. include:: programming-submission-policy-configuration.inc
```

Comment:
I know you did not start this naming convention, however, may I ask you to rename the `.inc` file to `.inc.rst` for better Highlighting on GitHub? Or is there a reason why this would not be a good idea?
```suggestion
.. include:: programming-submission-policy-configuration.inc.rst
```

Reply:
I was not aware that this is possible, I have changed it, thanks for the hint 👍